File: app/src/main/java/com/example/myapplication/ui/hearts/HeartsViewModel.java
package com.example.myapplication.ui.hearts;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class HeartsViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public HeartsViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is hearts fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/example/myapplication/ui/hearts/HeartsViewModel.kt
package com.example.myapplication.ui.hearts

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class HeartsViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is hearts Fragment"
    }
    val text: LiveData<String> = _text
}